config/settings/test.yml

```diff
@@ -1,2 +1,9 @@
 workflow:
   url: 'http://workflow.example.com/workflow'
+
+sdr:
+  local_workspace_root: /dor/workspace
+  abbyy:
+    # Windows-style paths for the ABBYY machine, referenced in XML tickets
+    remote_output_path: E:\AbbyyShare\sdr-ocr-qa\OUTPUT
+    remote_result_path: E:\AbbyyShare\sdr-ocr-qa\RESULTXML
```

Comment:
this is fine if it doesn't affect other tests, but just to note that if you want to get different settings for the context of a particular test, you can always do something like this without having to touch the actual config files:

https://github.com/sul-dlss/common-accessioning/blob/main/spec/lib/preserved_file_uris_spec.rb#L68-L71
https://github.com/sul-dlss/common-accessioning/blob/main/spec/robots/dor_repo/ocr/xml_ticket_create_spec.rb#L25-L29